.travis.yml

```diff
@@ -12,10 +12,17 @@ branches:
   only:
     - master
 
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+  - npm run webdriver-manager-update
+  - npm run webdriver-manager-start &> /dev/null &
+  - sleep 3 # Give selenium some time to start
 script:
   # DEV: Currently our test suite runs locally only
   #   This should be resolved in #19
-  - npm run lint
+  - npm run test-browserstack
```

Comment:
Due to the complex nature of Travis CI, I think we will still want some level of testing for PRs. The best we can do is linting so can we update the logic to be:

```bash
if test "$TRAVIS_PULL_REQUEST" == "false"; then npm run test-browserstack; else npm run lint; fi
```

It should be preceeded by a comment describing why we do it, for example:

```yml
# If we are on a pull request, avoid running Selenium tests
#    They will fail due to security restrictions on our credentials to prevent leaking
#    See #19 for more info
- "if test \"$TRAVIS_PULL_REQUEST\" == \"false\"; then npm run test-browserstack; else npm run lint; fi"
```